generator/restriction_dumper.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "std/fstream.hpp"
+#include "std/string.hpp"
+
+class RelationElement;
+
+namespace routing
+{
+class RestrictionDumper
+{
+  ofstream m_stream;
```

Comment:
Up to you, but usually private section is declared explicitly, at the end of the class.